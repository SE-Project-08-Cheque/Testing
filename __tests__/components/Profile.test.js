// import dependencies
import React from "react";

// import react-testing methods
import {cleanup } from "@testing-library/react";

// add custom jest matchers from jest-dom
import "@testing-library/jest-dom/extend-expect";

// import react dom
import ReactDOM from "react-dom";

// import renderer for take snapshots
import renderer from "react-test-renderer";

// import the component for testing
import Profile from "../../components/Profile";

afterEach(cleanup);


it("renders without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(
    <Profile />,
    div
  );
});

it("matches snapshot", () => {
    const tree = renderer.create(
      <Profile/>
    );
  
    expect(tree).toMatchSnapshot();
  });
  