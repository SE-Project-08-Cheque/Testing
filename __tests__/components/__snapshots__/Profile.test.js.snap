// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="center-to-screen"
>
  <div
    className="col-md-6"
  >
    <div
      className="jumbotron"
    >
      <h1>
        Profile
      </h1>
      <p>
        Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters, as opposed to using 'Content here, content here', making it look like readable English. Many desktop publishing packages and web page editors now use Lorem Ipsum as their default model text, and a search for 'lorem ipsum' will uncover many web sites still in their infancy. Various versions have evolved over the years, sometimes by accident, sometimes on purpose (injected humour and the like).
      </p>
      <div
        style={
          Object {
            "color": "#0056b3",
          }
        }
      />
    </div>
  </div>
</div>
`;
